v23.1/architecture/transaction-layer.md

```diff
@@ -155,60 +162,60 @@ The transaction record for a committed transaction remains until all its write i
+CockroachDB manages concurrency control using a per-node, in-memory lock table. This table holds a collection of locks acquired by in-progress transactions, and incorporates information about write intents as they are discovered during evaluation. For more information, see the [Concurrency control](#concurrency-control) section.
 
 #### Resolving write intents
 
-Whenever an operation encounters a write intent for a key, it attempts to "resolve" it, the result of which depends on the write intent's transaction record:
+Whenever a request encounters a write intent for a key, it attempts to "resolve" it, the result of which depends on the write intent's transaction record:
 
-- `COMMITTED`: The operation reads the write intent and converts it to an MVCC value by removing the write intent's pointer to the transaction record.
+- `COMMITTED`: The request reads the write intent and converts it to an MVCC value by removing the write intent's pointer to the transaction record.
 - `ABORTED`: The write intent is ignored and deleted.
-- `PENDING`: This signals there is a [transaction conflict](#transaction-conflicts), which must be resolved.
-- `STAGING`: This signals that the operation should check whether the staging transaction is still in progress by verifying that the transaction coordinator is still heartbeating the staging transaction’s record. If the coordinator is still heartbeating the record, the operation should wait. For more information, see [Parallel Commits](#parallel-commits).
+- `PENDING`: This signals there is a [lock conflict](#transaction-conflicts), which must be resolved.
+- `STAGING`: This signals that the request should check whether the staging transaction is still in progress by verifying that the transaction coordinator is still heartbeating the staging transaction’s record. If the coordinator is still heartbeating the record, the request should wait. For more information, see [Parallel Commits](#parallel-commits).
 - _Record does not exist_: If the write intent was created within the transaction liveness threshold, it's the same as `PENDING`, otherwise it's treated as `ABORTED`.
 
 ### Concurrency control
 
  The *concurrency manager* sequences incoming requests and provides isolation between the transactions that issued those requests that intend to perform conflicting operations. This activity is also known as [concurrency control](https://en.wikipedia.org/wiki/Concurrency_control).
 
-The concurrency manager combines the operations of a *latch manager* and a *lock table* to accomplish this work:
+The concurrency manager combines the tasks of a *latch manager* and a *lock table* to accomplish this work:
```

Comment:
Changed "operations" here to "tasks", it seemed to me here that "operation" was being used very generally for the jobs/tasks of the latch manager and lock table. The change here is to try and avoid using operation for anything other than what's happening with operations/requests.